library.properties

```diff
@@ -0,0 +1,9 @@
+name=Meshtastic
+version=0.0.1
+author=Mike Schiraldi
```

Comment:
I'd set this to `author=Meshtastic` as it will appear as an "Official Library" vs something 3rd party.

Reply:
Done